src/test/java/org/openelisglobal/dictionary/form/DictionaryFormTest.java

```diff
@@ -0,0 +1,76 @@
+package org.openelisglobal.dictionary.form;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+
+import java.util.List;
+import org.junit.Before;
+import org.junit.Test;
+import org.openelisglobal.BaseWebContextSensitiveTest;
+import org.openelisglobal.dictionary.service.DictionaryService;
+import org.openelisglobal.dictionary.valueholder.Dictionary;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class DictionaryFormTest extends BaseWebContextSensitiveTest {
+
+    @Autowired
+    private DictionaryService dictionaryService;
+
+    @Before
+    public void setUp() throws Exception {
+        executeDataSetWithStateManagement("testdata/dictionary.xml");
+    }
+
+    @Test
+    public void GetDictionariesByCategoryId_shouldReturnDictionaries() {
```

Comment:
this looks like the same as https://github.com/DIGI-UW/OpenELIS-Global-2/blob/e1217cdeb477c5702178c2ad22126f5508821c00/src/test/java/org/openelisglobal/dictionary/service/DictionaryServiceTest.java#L52